docs/config.md

```diff
@@ -579,6 +579,17 @@ as follows:
 signing.backends.ssh.allowed-signers = "/path/to/allowed-signers"
 ```
 
+### Using a filepath
```

Comment:
nit: `#### ...` (4-th level header)?
It's specific to SSH backend.

I think it's also good to just inline the example as a comment
```toml
key = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGj+J6N6SO+4P8dOZqfR1oiay2yxhhHnagH52avUqw5h"
# or file path
# key = "/home/me/.ssh/id_for_signing.pub"
```

Reply:
I inlined both yours and @necauqua's comment